test/jasmine/tests/polar_test.js

```diff
@@ -232,8 +232,21 @@ describe('Test relayout on polar subplots:', function() {
         var fig = Lib.extendDeep({}, require('@mocks/polar_scatter.json'));
 
         Plotly.plot(gd, fig).then(function() {
+            expect(gd._fullLayout.polar._subplot.viewInitial['radialaxis.range'])
+                .toBeCloseToArray([0, 11.225]);
+
             return Plotly.relayout(gd, 'polar.radialaxis.type', 'log');
         })
+        .then(function() {
+            expect(gd._fullLayout.polar._subplot.viewInitial['radialaxis.range'])
+                .toBeCloseToArray([-0.53, 1.158]);
+
+            return Plotly.relayout(gd, 'polar.radialaxis.type', 'linear');
+        })
+        .then(function() {
+            expect(gd._fullLayout.polar._subplot.viewInitial['radialaxis.range'])
+                .toBeCloseToArray([0, 11.225]);
+        })
```

Comment:
Good catch! Just for my peace of mind so I don't have to trust that `viewInitial` does the right thing, can you also `expect` that `fullLayout.polar.radialaxis.range` matches `viewInitial[...range]`?

Reply:
Good call. Fixed in https://github.com/plotly/plotly.js/pull/2200/commits/473643cf8dfb12f1543bd8dcb01ca00c568f6398